cli/medperf/tests/test_utils.py

```diff
@@ -340,7 +340,7 @@ def test_approval_prompt_returns_approved_boolean(mocker, ui, input_val):
     assert approved == (input_val in "yY")
 
 
-@pytest.mark.parametrize("dict_with_nones", rand_l(0, 100, 5), indirect=True)
+@pytest.mark.parametrize("dict_with_nones", [32, 7, 90], indirect=True)
```

Comment:
Same argument as the `datasets` fixture. `dict_with_nones` also contains randomness.